packages/bot-api/src/MessageHandler.ts

```diff
@@ -51,6 +52,14 @@ export abstract class MessageHandler {
     }
   }
 
+  public async getConversation(conversationId: string): Promise<Conversation> {
+    return (await this.account!.service!.conversation.getConversations([conversationId]))[0];
```

Comment:
```suggestion
    return this.account!.service!.conversation.getConversation(conversationId);
```

Reply:
Property 'getConversation' does not exist on type 'ConversationService'. 😈 @lipis please stick to `await` code from before.